### DISCUSSION TITLE
About Linux and MacOS

### DISCUSSION THREAD

**Discussion by maximmax42** [maximmax42/Discord-CustomRP#52] [General]
**TL;DR There won't be Linux or MacOS version (for the foreseeable future), but you can try using Wine/Lutris. Or you can use [another tool by ThatOneCalculator](https://drpcm.t1c.dev/).**

Now for the longer explanation. The app is written in C# using a library called Windows Forms. As the name suggests, it's a Windows only library. There are cross-platform libraries in C#, but they all follow an MVVM pattern that I'm not familiar with. In the future I might move the project to one of those, because in addition to not being cross-platform, WinForms is a pain in the ass to customize properly. Implementing dark mode was a huge pain, and some of the elements aren't allowed to be colored in properly. But I can't give any promises on when I would be able to migrate the app.

**Comment by M4rYL** [comment]
Wine doesn't work on this. The app crashes on startup with Wine error.
PoP!OS, 22.04, everything up to date, 01/August/2022.

**Reply by maximmax42** [reply]
Works for some people, doesn't for others. You can also try Lutris.

**Comment by kxws** [comment]
Wine works for me, just cant detect discord. says "connection failed (is discord running).
